### PR TITLE
Add turning counts routes

### PR DESCRIPTION
**Pull request by runnersaw**
Adds turning counts routes to cloud_api. Should go in with [this SantosCloud PR](https://github.com/santosfamilyfoundation/SantosCloud/pull/101).

### CODE REVIEW

File: application/cloud_api.py [hunk 1]
```diff
@@ -409,6 +412,11 @@ def results(self, identifier, file_path, ttc_threshold = None):
         if not s:
             return (s, err)
 
+        s, err = self.turningCounts(identifier,\
+                    file_path)
+        if not s:
+            return (s, err)
```
Comment (dcelik) [inline application/cloud_api.py]: Same here.

File: application/cloud_api.py [hunk 2]
```diff
@@ -32,6 +32,9 @@ def set_url(self, ip_addr, port=8888):
         self.server_addr = protocol + addr + ':{}/'.format(port)
 
     def parse_error(self, r):
+        content_type = r.headers['content-type']
+        if 'application/json' not in content_type:
+            return (True, None)
```
Comment (dcelik) [inline application/cloud_api.py]: This should be a 3-tuple. See #102.

File: application/cloud_api.py [hunk 3]
```diff
@@ -620,6 +628,40 @@ def speedDistribution(self, identifier, file_path):
...
+            os.remove(path)
+
+        if not os.path.exists(file_path):
+            os.makedirs(file_path)
+
+        with open(path, 'wb') as f:
+            print('Dumping "{0}"...'.format(path))
+            for chunk in r.iter_content(chunk_size=2048):
+                if chunk:
+                    f.write(chunk)
```
Comment (dcelik) [inline application/cloud_api.py]: Use the write_to_path helper function. See #102

**Review by dcelik — CHANGES_REQUESTED** [review]
Make changes to match changes in #102. Other than that looks good. A rebase off of new dev could fix this.

**Review by dcelik — APPROVED** [review]

**Comment by runnersaw** [comment]
Also includes using `pathos` multiprocess rather than multiprocessing in order to fix Windows issues. Appears to speed up download and video rendering on all platforms!

**Pull request merged**